week2/algoORgoal/index.css

```
@@ -0,0 +1,182 @@
+/* Global Styels */
+button {
+  background: none;
+  border: none;
+  padding: 0;
+  cursor: pointer;
+  outline: inherit;
+}
+
+* {
+  font-family: Inter;
+}
+
+.filters {
+  position: sticky;
+  top: 0;
+  background-color: white;
+  width: 100%;
+  border-bottom: 1px #d9d9d9 solid;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 12px 16px;
+  box-sizing: border-box;
+}
+
+.filters__area {
+  display: flex;
+  align-items: center;
+  column-gap: 6px;
+}
+
+.filters__area-name {
+  color: black;
+  font-size: 18px;
+  font-weight: 800;
+  line-height: 28px;
+}
+
+.filters__icon-container {
+  display: flex;
+  column-gap: 16px;
+  align-items: center;
+}
+
+filters__area-icon {
+}
+
+.filters__search {
+}
+
+.filters__menu {
+}
+
+.filters__notifications {
+}
+
+.tap-bar {
+  position: sticky;
+  width: 100%;
+  bottom: 0;
+  display: flex;
+  justify-content: space-around;
+  border-top: 1;
+  border: 1px #d9d9d9 solid;
+  padding: 6px 0px;
+  background-color: white;
+}
+
+.tap-bar__button {
+}
+
+.tap-bar__button-icon {
+}
+
+.tap-bar__button-text {
+  color: black;
+  font-size: 10px;
+  font-family: Inter;
+  font-weight: 800;
+  line-height: 14px;
+}
+
+.products {
+}
+
+.products__item {
+  display: flex;
+  align-items: stretch;
+  column-gap: 16px;
+  /* gap: 0px 16px; */
+  padding: 16px 16px;
+  border-bottom: 1px #d9d9d9 solid;
+}
+
+.products__item:last-child {
+  border-bottom: none;
+}
+
+.products__item-description {
+  display: flex;
+  flex-direction: column;
+  row-gap: 4px;
+}
+
+.products__item-name {
+}
+
+.products__item-detail {
+  color: #8c8c8c;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 16px;
+}
+
+.products__item-img {
+  width: 110px;
+  height: 110px;
+  border-radius: 4px;
+}
+
+.products__item-area {
+}
+
+.products__item-uploaded {
+}
+
+.products__item-price {
+  color: #ff7e36;
+  font-size: 15px;
+  font-weight: 700;
+  line-height: 22px;
+  font-family: Inter;
+}
+
+.products__item-texts {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.products__item-feedback {
+  display: flex;
+  width: 100%;
+  justify-content: end;
+  color: #8c8c8c;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 16px;
+  column-gap: 4px;
+}
```

```suggestion
.products__item-feedback {
  display: flex;
  width: 100%;
  justify-content: end;
  align-items: center;
  color: #8c8c8c;
  font-size: 12px;
  font-weight: 400;
  line-height: 16px;
  column-gap: 4px;
}
```
`align-items: center;`로 flex item들을 중앙 정렬 시키면 더 좋을거 같아요!